types/src/v0/v0_1/instance_state.rs

```diff
@@ -8,7 +8,7 @@ use crate::{v0_3::ChainConfig, Timestamp};
 #[serde(rename_all = "snake_case")]
 pub enum UpgradeType {
     Fee { chain_config: ChainConfig },
-    Marketplace {},
+    Marketplace { chain_config: ChainConfig },
```

Comment:
why are we passing chain config here

Reply:
oh nvm I think its `bid_recipient`

Reply:
humm but I guess we should add header in there as well